test/filters/browser_tests-macos.filter

```diff
@@ -6,4 +6,4 @@
 ##
 
 # Browser tests run extremely slowly on MacOS, disable them all for now.
--*
```

Comment:
please revert this

Reply:
Argh, that's a bad inadvertent commit. Thanks for catching it. Fixed